frontend/simple/js/state.js

```diff
@@ -57,13 +60,19 @@ const mutations = {
   },
   deleteMessage (state, hash) {
     let mailboxContract = store.getters.mailboxContract
-    let index = mailboxContract && mailboxContract.messages.findIndex(msg => msg.hash === hash)
-    if (index > -1) { mailboxContract.messages.splice(index, 1) }
+    let index = mailboxContract &&
+      mailboxContract.messages.findIndex(msg => msg.hash === hash)
+    if (index > -1) {
+      mailboxContract.messages.splice(index, 1)
+    }
   },
   markMessageAsRead (state, hash) {
     let mailboxContract = store.getters.mailboxContract
-    let index = mailboxContract && mailboxContract.messages.findIndex(msg => msg.hash === hash)
-    if (index > -1) { mailboxContract.messages[index].read = true }
+    let index = mailboxContract &&
+      mailboxContract.messages.findIndex(msg => msg.hash === hash)
+    if (index > -1) {
+      mailboxContract.messages[index].read = true
+    }
```

Comment:
Let's stick to 1-liners for these `if`s, or if you don't like that, then convert them to &&'s: `index > -1 && mailboxContract.messages[index].read = true` (and above). 

Keeping the line count down is somewhat of a religious practice for this codebase, but yes I understand that it must be balanced against having lines that are too long. I think a 100 line width margin is fine for the project, given that the 80 width is probably based on ancient terminal screens, and today people have much larger displays. Keeping it at 100 keeps the line count down, while being reasonable in terms of line width, IMO at least.